### ISSUE TITLE
Compute Errors - CN/Half

### ISSUE THREAD

**Issue by Bathmat** [xmrig/xmrig-nvidia#231]
I'm seeing some compute errors with CN/Half algo. 
```
OS: Win7 (x64)
Driver: 391.01
GPUs: GTX-970, 1050
Compiled with CUDA 9.1
```
I did not see any compute errors with this setup/configuration/CUDA version on any previous algo.

**Comment by Bathmat** [comment]
I did a quick test and it looks like this is fixed with https://github.com/xmrig/xmrig-nvidia/pull/232

**Comment by xmrig** [comment]
Fixed in v2.9.3.
Thank you.
